sklearn/model_selection/_split.py

```diff
@@ -406,8 +420,11 @@ class KFold(_BaseKFold):
     RepeatedKFold: Repeats K-Fold n times.
     """
 
-    def __init__(self, n_splits=3, shuffle=False,
+    def __init__(self, n_splits=None, shuffle=False,
```

Comment:
I thought we're gonna use ``'warn'`` from now on?

Reply:
You want to replace all None by "warn"? Fine with me.

Reply:
@amueller for `n_splits` only or `cv` as well ?